[Flutter] Documentation updates (#1404)
WalletConnect/walletconnect-docs: docs/web3wallet/wallet-usage.mdx

To reject the request, pass in an error code and reason. They can be found [here](https://specs.walletconnect.com/2.0/specs/clients/sign/error-codes).

> To reject a session use:

Reviewer: ```suggestion
To reject a session:
```

</PlatformTabItem>
<PlatformTabItem value="flutter">

> To handle a session request, sucha as `personal_sign`, you would have to register a request handler as mentioned in poin 2 of Namespace Builder section.

Reviewer: ```suggestion
To handle a session request, such as `personal_sign`, you would have to register a request handler as mentioned in point 2 of Namespace Builder section.
```

> You would have to register a handler for every chain and method you want to support. So let's say your wallet support `eip155:1` and `eip155:137`, this would a simply code for it:

Reviewer: ```suggestion
You would have to register a handler for every chain and method you want to support. So let's say your wallet supports `eip155:1` and `eip155:137`. This would translate to:
```

```dart
await web3Wallet.disconnectSession(
  topic: session.topic,
  reason: Errors.getSdkError(Errors.USER_DISCONNECTED),
);
```

> Using `disconnectSession()` alone will make the pairing toppic to persist so it can be re-used until it expires. If you want to disconnect (remove) the pairing topic as well you would have add another call as follows:

Reviewer: ```suggestion
Using `disconnectSession()` alone will make the pairing topic persist, i.e, it can be re-used until it expires. If you want to disconnect (remove) the pairing topic as well you would have add another call as follows:
```

</PlatformTabItem>
<PlatformTabItem value="flutter">

#### Supporting session events

> In order to support session events, such as `chainChanged` or `accountChanged` your would have first to register an event emitter for such events for every chain you want to emit an event for similar to, similar to request handlers.

Reviewer: ```suggestion
In order to support session events, such as `chainChanged` or `accountChanged`, you would have to to register an event emitter for such events, for every chain you want to emit an event for (similar to request handlers).
```

```dart
final supportedChains = ['eip155:1', 'eip155:137'];
const supportedEvents = ['chainChanged', 'accountChanged'];
for (var chainId in supportedChains) {
  for (var event in supportedEvents) {
    _web3Wallet.registerEventEmitter(
      chainId: chainId,
      event: event,
    );
  }
}
```

> And to emit an event you just call `emitSessionEvent()` as follows:

Reviewer: ```suggestion
And to emit an event, call `emitSessionEvent()` as follows:
```
